src/PaymentGateways/Gateways/PayPalCommerce/payPalCommerceGateway.tsx

```diff
@@ -528,7 +528,10 @@ import createSubscriptionPlan from './resources/js/createSubscriptionPlan';
          */
         Fields() {
             const {isRecurring} = window.givewp.form.hooks.useFormData();
-            submitButton = window.givewp.form.hooks.useFormSubmitButton();
+
+            useEffect(() => {
+                submitButton = window.givewp.form.hooks.useFormSubmitButton();
+            }, []);
```

Comment:
Let's add an `unreleased` tag about this change since it is already using the tags to track changes. Also, I noticed that the `showOrHideDonateButton` and other functions don't have the tags, and I don't want to get stuck on anything because of it, but if we could add the proper tags to the places where it's missing, it would be great!

Reply:
sounds good! resolved [5403db3](https://github.com/impress-org/givewp/pull/7891/commits/5403db3727be0319ac173c0d7a1d4b4cde4a0150)